Fix coala/landing-frontend#189

raghavmittal101 opened the pull request:
Added `overflow-x` property to `.coafile` .

raghavmittal101 committed:
Fix coala/landing-frontend#189

Added `overflow-x` property to `.coafile` .

hemangsk commented:
Please get assigned to the issue you are fixing, for that you can go on gitter and say Hello World, you will get an email invite to join coala org :)

raghavmittal101 commented:
Thank you for kind help. I have followed the steps you suggested and assigned this issue to myself.

hemangsk commented on coala.css:
```diff
@@ -773,6 +773,10 @@ a.chip i {
+
 .coala-online .coafile {
   background: #263238;
   white-space: pre-line;
```
overflow-x: auto should go inside the ```.coala-online .coafile```  selector

raghavmittal101 committed:
stylee.css : add autoscroll property to container

The `overflow-x: auto` property in `.coafile, .coala-online` selector
ensures horizontal scrolling when there are lot of bears.

Fixes https://github.com/coala/landing-frontend/issues/189

raghavmittal101 committed:
coala.css: add autoscroll property to container

By adding `autoscroll-x: auto` property to `.coafile`, `.coala-online`, I ensure horizontal scrolling inside the container when there are lot of bears.

Fixes https://github.com/coala/landing-frontend/issues/189

hemangsk commented:
@raghavmittal101 awesome, just one more request to squash both of the commits into one, and to follow the commit message guidelines on https://coala.io/commit

raghavmittal101 committed:
coala.css: Make textarea unresizable along horizontal

Horizontal resizable property is disabled by adding `resize: vertical` to `textarea.pre`. This line ensures that textarea will only be vertically resizable by hand.

Fixes https://github.com/coala/landing-frontend/issues/190

raghavmittal101 committed:
Revert "coala.css: Make textarea unresizable along horizontal"

This reverts commit 29f56de982cb90f8a03e5ad88ac77b07188b9fe6.

Arsfiqball commented:
@raghavmittal101 you need to squash your commit.

raghavmittal101 committed:
Merge branch 'master' of https://github.com/raghavmittal101/coalaCSS

raghavmittal101 commented:
I have created a new pull request from another branch here https://github.com/coala/coalaCSS/pull/16. Please consider that one. Sorry for creating all this mess and wasting precious time of the community members.

yukiisbored commented:
Closing for newer PR https://github.com/coala/coalaCSS/pull/16. Also, @raghavmittal101 here's a tip GitHub (and GitLab as well) doesn't care about the branch at all it just only see the name, You can just copy your work over to another branch, destroy the old one that's being used in the PR, create a new one, cherry pick the commits from the other branch (you can modify them if you wish!), and do a force push to the same branch name. et Voila! no need to create new pull requests!

Feel free to ask us questions about git and the tools that we use on our gitter channels :)

raghavmittal101 commented:
@yukiisbored and @Arsfiqball Thank you for helping m. :) . I am using github since a couple of years, but never worked with opensource organizations like coala. Its a nice experience. :sweat_smile:
